src/Sylius/Bundle/CustomerBundle/composer.json

```diff
@@ -62,6 +62,9 @@
     "extra": {
         "branch-alias": {
             "dev-main": "2.0-dev"
+        },
+        "symfony": {
+            "require": "7.0.*"
```

Comment:
Do you need to specify this on every composer? At least some of them missing it, e.g. Channel, Attribute, Api, Locale, Order, Payment bundles

Reply:
Right, I'll do that on missing packages.